src/main/java/net/neoforged/neoforge/common/ModConfigSpec.java

```diff
@@ -244,25 +271,21 @@ private int correct(UnmodifiableConfig spec, CommentedConfig config, LinkedList<
         return count;
     }
 
-    private boolean stringsMatchIgnoringNewlines(@Nullable Object obj1, @Nullable Object obj2) {
-        if (obj1 instanceof String string1 && obj2 instanceof String string2) {
-            if (string1.length() > 0 && string2.length() > 0) {
-                return string1.replaceAll("\r\n", "\n")
-                        .equals(string2.replaceAll("\r\n", "\n"));
-
-            }
+    private boolean stringsMatchIgnoringNewlines(@Nullable String string1, @Nullable String string2) {
```

Comment:
Ignoring wouldn't be technically correct, I'd name it normalizingNewLines instead